include/tsid/math/fwd.hpp

```diff
@@ -20,8 +20,13 @@
 
 #include <Eigen/Core>
 
-#define EIGEN_MALLOC_ALLOWED Eigen::internal::set_is_malloc_allowed(true);
-#define EIGEN_MALLOC_NOT_ALLOWED Eigen::internal::set_is_malloc_allowed(false);
+#ifdef EIGEN_RUNTIME_NO_MALLOC
```

Comment:
and in 
https://github.com/stack-of-tasks/tsid/blob/master/include/tsid/solvers/solver-HQP-eiquadprog-rt.hxx#L174
https://github.com/stack-of-tasks/tsid/blob/master/include/tsid/solvers/solver-HQP-eiquadprog-rt.hxx#L150